src/common.jl

```diff
@@ -5,6 +5,20 @@
     POLYNOMIAL = 2
 end
 
+@enum AngleUnit begin
+    DEGREES
+    RADIANS
+end
+
+@enum BusType begin
+    REF
+    ISOLATED  # TODO DT: this was in a matpower file, verify
```

Comment:
There was some discussion on our support for islanded buses. @jd-lara I don't recall where we landed. If we don't include the `ISOLATED` bus type in the parser, does the data just get skipped?

Reply:
It will currently throw an error because it can convert "isolated" to a type.  I could make it parse to UNKNOWN.